docker/dev/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:lts
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+COPY . .
+RUN npm install
+
+# build dist
+RUN npm run build
+
```

Comment:
The `sqlite3` package needs to be installed manually.

Reply:
Do you just install it with `sudo apt-get install sqlite3` and do you also start the sqlite3 server

Reply:
It's the npm package sqlite3